src/PkgTaskListWidget.cc

```diff
@@ -110,10 +110,7 @@ bool PkgTaskListWidgetItem::operator<( const QListWidgetItem & otherListWidgetIt
     {
         return ( this->serial() < other->serial() );
     }
-    else
-    {
-        return QListWidgetItem::operator<( otherListWidgetItem );
-    }
+    return QListWidgetItem::operator<( otherListWidgetItem );
```

Comment:
Same as above: This silently implies that the `if ()` branch is the exceptional case while the other one (which was the `else` branche) is the normal control flow. No, it's not; the `else` branch is the exceptional case. And again, the compiler will complain if there were any way that neither branch is executed, so the function might not return a value. It's certain that it does.